.github/workflows/ci-shell-scripts.yml

```
@@ -0,0 +1,29 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "*.*.x"
+    paths:
+      - "bin/**"
```

This workflow should run when modified.

```suggestion
      - ".github/workflows/ci-shell-scripts.yml"
      - "bin/**"
```